docker-compose.yml

```diff
@@ -29,6 +30,8 @@ services:
     volumes:
       - .:/usr/src/app
       - app_node_modules:/usr/src/app/node_modules
+      - app_build:/usr/src/app/build
```

Comment:
One option for a better way: any assets that are built by the repo but are ultimately system independent, could be built before the container build cycle and then copied into the container.

One thing we'd have to be sensitive about is how to handle any CDN paths in the built assets.